src/components/Input/Input.js

```diff
@@ -1,64 +1,34 @@
 import React, {Component, PropTypes} from 'react';
-import Field from '../Field/Field';
-import $ from 'jquery';
 import classNames from 'classnames';
 
-class Input extends Component {
+export default class Input extends Component {
   static propTypes = {
+    className: PropTypes.string,
     dataContent: PropTypes.string,
     defaultValue: PropTypes.string,
     iconClass: PropTypes.string,
-    label: PropTypes.string,
-    name: PropTypes.string,
-    onChange: PropTypes.func,
     ref: PropTypes.string,
-    small: PropTypes.string,
-    transparent: PropTypes.bool,
-    type: PropTypes.string,
-    value: PropTypes.string,
   };
 
   static defaultProps = {
-    ref: 'popper',
     type: 'text',
   };
 
-  componentWillMount() {
-    this.elm = $(React.findDOMNode(this.refs[this.props.ref]));
-    if (this.props.dataContent) {
-      this.elm.popup({
-        on: 'focus',
-      });
-    }
-  }
```

Comment:
There is no more built in popup functionality, this should be accomplished with a popup from the outside.